src/plugins/news/components/create-article-modal/index.js

```diff
@@ -17,10 +17,29 @@ import trimStart from 'lodash-es/trimStart.js';
 const DATED_SLUG_PATTERN = /^\d{4}\/\d{2}\/\d{2}\/[a-z0-9-]+$/;
 const isValidDatedSlug = value => isValidSlug(value, true) && DATED_SLUG_PATTERN.test(trimStart(value));
 
+const parseDatedSlug = (value) => {
+  let ending = '';
+  if (value && (value.endsWith(' ') || value.endsWith('/') || value.endsWith('-'))) {
+    ending = value[value.length - 1];
+  }
+  const slug = createSlug(value.toLowerCase(), true);
+  return slug ? slug + ending : value;
+};
+
+const slugValidator = (value) => {
+  if (!value) return undefined;
+  if (value.includes('/')) {
+    if (!isValidDatedSlug(value)) {
+      return 'Expected format YYYY/MM/DD/some-title-here.';
+    }
+  }
+  const trimmedValue = value?.trim().replace(/\s+/g, '-') ?? value;
+  return isValidSlug(trimmedValue, true) ? undefined : 'Only use letters, numbers and dashes.';
+};
```

Comment:
@meghedi why is this so different from the other one? shouldnt they be essentially the same?